ipaplatform/redhat/authconfig.py

```diff
@@ -133,6 +122,8 @@ def configure(self, sssd, mkhomedir, statestore, sudo=True):
         if sudo:
             cmd.append("with-sudo")
         cmd.append("--force")
+        cmd.append("-b")
```

Comment:
`-b` is short-cut for `--backup`, why using both of them?

Reply:
Good catch, fixed.